core/src/sigverifier/bls_sigverifier.rs

```diff
@@ -39,76 +61,74 @@ impl SigVerifier for BLSSigVerifier {
         packet_batches: Vec<PacketBatch>,
     ) -> Result<(), SigVerifyServiceError<Self::SendType>> {
         // TODO(wen): just a placeholder without any batching.
-        let mut packet_received = 0;
-        let mut bls_messages_sent = 0;
-        let mut bls_messages_malformed = 0;
-        let mut sent_failed = 0;
+        let mut stats = BLSSigVerifierStats::new();
         packet_batches.iter().for_each(|batch| {
             batch.iter().for_each(|packet| {
-                packet_received += 1;
+                stats.received += 1;
                 match packet.deserialize_slice::<BLSMessage, _>(..) {
                     Ok(message) => match self.sender.try_send(message) {
                         Ok(()) => {
-                            bls_messages_sent += 1;
+                            stats.sent += 1;
                         }
                         Err(TrySendError::Full(_)) => {
                             warn!("BLS message channel is full, dropping message");
```

Comment:
we should use the existing error handling by setting:
`SendType = String`
And then returning the `SigVerifyServiceError` variant for these by using `SendPktsError` for the transient errors, and the `Send` error for the disconnected situation.

Reply:
Changed SendType. I'm confused, the code is not using `solana_streamer`, why should it return `SendPktsError`?

I checked the error handling currently in sigverifier, it doesn't do much.
https://github.com/anza-xyz/alpenglow/blob/8e3a696f163f6c876921b49e227b8bcc619f511d/core/src/sigverify_stage.rs#L336
One error just makes the whole send_packets() end,
https://github.com/anza-xyz/alpenglow/blob/8e3a696f163f6c876921b49e227b8bcc619f511d/core/src/sigverify_stage.rs#L411
Then it just exits the whole loop. Not much metrics collecting either, it just logs or ignores the returned error.

This makes sense when we used unbounded channel, since we would never see SendError::Full. Now we use bounded channel, we need to handle TrySendError::Full.